.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,56 @@
+name: DualGovernance PR tests
+
+permissions:
+  contents: read
+  pull-requests: write
+  security-events: write
+
+on:
+  pull_request:
+    branches: ['**']
```

Comment:
Since regression tests are run against the deployed version of the protocol, it may be sufficient to run them only on merges to the develop and main branches.